src/app/public/modules/modal/modal.module.ts

```diff
@@ -67,16 +65,14 @@ import {
     SkyModalScrollShadowDirective
   ],
   providers: [
-    MutationObserverService,
-    SkyAppWindowRef,
     SkyThemeService
```

Comment:
Should we be providing the `SkyThemeService` in this module? Might be good to make this a checklist item for all libraries...

Reply:
This was fixed when we merged the latest from `5.0.0-next`.